mod/projects/set/abstract/project_scope.rb

```diff
@@ -0,0 +1,105 @@
+# module for project-scope-definining cards
+# (+company, +metric, and +year on projects)
```

Comment:
I'm not sure this is the best naming, but the idea was that there was a lot of duplicate code in +metric, +company, and +year, and I wanted to consolidate rather than add more.